docs/book/routing.md

```diff
@@ -159,8 +159,8 @@ The `Hostname` route attempts to match the hostname registered in the request
 against specific criteria. Typically, this will be in one of the following
 forms:
 
-* `subdomain.domain.tld`
-* `:subdomain.domain.tld`
+- `subdomain.domain.tld`
```

Comment:
why commit message is  "PHP 8.4 support added" and the change is unrelated to `.md` file? We don't use squash on merge so I think it would be better to have proper message if this is a blocker for latest CI.

Reply:
An example of good commit messages can be found in this pull request: https://github.com/laminas/laminas-feed/pull/89

Reply:
I've updated the commit message.
@samsonasik I'll take care of it next time. Thanks @froschdesign for the reference.